src/universal/components/ReflectionGroup/ReflectionGroup.js

```diff
@@ -62,21 +60,23 @@ class ReflectionGroup extends Component<Props, State> {
 
   setTopCardRef = (c) => {
     this.topCardRef = c;
+    this.forceUpdate();
```

Comment:
need to manually calculate the height of a group to reduce the big fat bottom margin. can't calculate it until it's rendered to the DOM.